test/sagemaker_tests/huggingface_pytorch/training/integration/sagemaker/test_smdp.py

```diff
@@ -90,7 +90,7 @@ def test_smdp_question_answering(ecr_image, instance_type, py_version, sagemaker
     with timeout(minutes=DEFAULT_TIMEOUT):
         estimator = HuggingFace(
             entry_point='run_qa.py',
-            source_dir='./examples/question-answering',
+            source_dir='./examples/pytorch/question-answering',
```

Comment:
Could you please change this to be:
```
source_dir = (
    "./examples/question-answering"
    if Version(transformers_version) < Version("4.6")
    else "./examples/pytorch/question-answering"
)
...
estimator = HuggingFace(
    ...
    source_dir=source_dir,
    ...
)
```
so that we can continue to use these tests with HF Transformer versions 4.5 as well as 4.6?